.woodpecker.yml

```diff
@@ -79,9 +79,11 @@ steps:
   cargo_machete:
     image: rustlang/rust:nightly
     commands:
+      # Install cargo binstall
       - wget https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz
```

Comment:
Not sure if there's a way to extract and run multiple lines in yaml... if anyone knows how to do this that'd be appreciated.